.ci/Jenkinsfile

```diff
@@ -129,9 +128,10 @@ pipeline {
 
 def runJob(testName, buildOpts = ''){
   def branch = env.CHANGE_ID?.trim() ? env.CHANGE_TARGET : env.BRANCH_NAME
-  def job
+  def jobName = "apm-integration-test-downstream/${branch}"
+  def buildObject
   try {
-    job = build(job: "apm-integration-test-downstream/${branch}",
+    buildObject = build(job: "apm-integration-test-downstream/${branch}",
```

Comment:
I guess the `jobName` variable is needed here

Reply:
```suggestion
    buildObject = build(job: jobName,
```